_posts/2023-03-22-cockpit-288.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Cockpit 288
+author: kkoukiou
+date: '2023-03-22'
+tags: cockpit, machines, podman
+slug: cockpit-288
+category: release
+summary: 'Metrics redesign, accounts page improvements, virtual machines cloud base editing, services page speed ups, chronyd supports timeservers, and esbuild bundler'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 288, cockpit-machines 286, and cockpit-podman 65:
+
+## Accounts: Show shell and home directory on an account's detail page
```

Comment:
Too long, the heading wraps. It's already clear that we are talking about Accounts here.

```suggestion
## Accounts: Show shell and home directory on an detail page
```